### PR TITLE
typo fix

### PR DESCRIPTION
**Pull request by jonyhy96**
typo fix
```release-note
None
```

### CODE REVIEW

**Comment by timstoop** [comment]
Recent changes in our requirements now demand a home key in the Chart.yaml. This should contain the URL to the upstream project. If you can add that in there as well, your PR can be accepted and you'll fix it for other PRs for this Chart as well.

**Comment by jonyhy96** [comment]
@timstoop Thanks for reply
Do you mean I should add a key "home" in Chart.yaml and the value is "https://github.com/kubernetes/charts/tree/master/stable/prometheus-postgres-exporter"?

**Comment by timstoop** [comment]
Indeed. I know it's not a problem that you caused, but it's easy for you to fix and once fixed, it's fixed for every PR for this Chart, so it would be appreciated.

**Comment by jonyhy96** [comment]
@timstoop Thanks for your patient and I’m glad to make contributions on that

**Comment by jonyhy96** [comment]
@timstoop I already add the home:url in the Chart.yaml 
Should I add it in each program's Chart.yaml?

**Comment by timstoop** [comment]
No, it's now throwing another error. I've seen it before, but not entirely sure what's going on. It seems like the image is writing in a read-only filesystem, but I cannot find a cause for that. I'll look into it later today. Thanks for your PR!

**Comment by jonyhy96** [comment]
@timstoop Thanks for your help and wish you have a good day!:D

**Comment by unguiculus** [comment]
cc @gianrubio

**Comment by jonyhy96** [comment]
@gianrubio 
The test still failed.

File: stable/prometheus-postgres-exporter/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,8 @@ apiVersion: v1
 appVersion: "0.4.4"
 description: A Helm chart for prometheus postgres-exporter
 name: prometheus-postgres-exporter
-version: 0.1.0
+version: 0.1.1
+home: https://github.com/kubernetes/charts/tree/master/stable/prometheus-postgres-exporter
```
Comment (gianrubio) [inline stable/prometheus-postgres-exporter/Chart.yaml]: home should be https://github.com/wrouesnel/postgres_exporter

**Comment by gianrubio** [comment]
@jonyhy96 the error in the tests are happening because we are mounting /etc and for certain reasons the e2e test is trying to mount /etc. Please replace the `/etc` mount to `/config` and add the arg below
```
          args:
            - -extend.query-path=/config
```

**Comment by jonyhy96** [comment]
I will close this PR and fix these mistakes in a new PR.
close it.

**Comment by jonyhy96** [comment]
@gianrubio 
Thanks for your help 
I restart a new PR [fix a typo and add home link #4669](https://github.com/kubernetes/charts/pull/4669)
I'm not quite understand how to replace the /etc mount to /config and add the arg.
Could you please specify the file where i should modify.
Tnaks for your generous help and your patience.
Wish you have a good day!:D

**Pull request closed without merging**